react/src/views/Box/components/BoxDetails.tsx

```diff
@@ -285,6 +285,12 @@ function BoxDetails({
                     border="2px"
                   >
                     {location.name}
+                    {location.defaultBoxState !== boxData.state && (
+                      <>
+                        {" "}
+                        - Boxes are &nbsp;<i> {location.defaultBoxState}</i>
```

Comment:
![2022-12-18-135851_618x547_scrot](https://user-images.githubusercontent.com/10617122/208299827-536dbbda-1301-4df1-b653-cc7fb6f92cbb.png)

The space between "Boxes are" and the state name could be smaller IMHO.

Reply:
Removing the space before `nbsp` looks better I find